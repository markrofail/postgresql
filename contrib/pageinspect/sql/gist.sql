CREATE TABLE gist_table (t box);
\copy gist_table from 'data/rect.data'

CREATE INDEX gist_idx ON gist_table USING gist (t);

SELECT gist_stats('gist_idx');
