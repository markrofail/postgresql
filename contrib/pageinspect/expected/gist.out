CREATE TABLE gist_table (t box);
\copy gist_table from 'data/rect.data'
CREATE INDEX gist_idx ON gist_table USING gist (t);
SELECT gist_stats('gist_idx');
               gist_stats                
-----------------------------------------
 Number of levels:          2           +
 Number of pages:           29          +
 Number of leaf pages:      28          +
 Number of tuples:          3406        +
 Number of invalid tuples:  0           +
 Number of leaf tuples:     3378        +
 Total size of tuples:      143516 bytes+
 Total size of leaf tuples: 142296 bytes+
 Total size of index:       237568 bytes+
 
(1 row)

